src/features/landing/components/AuthSection/ConnectedForm/ConnectedForm.tsx

```diff
@@ -22,12 +24,12 @@ function ConnectedForm(): React.ReactElement {
           },
         },
       });
-    } catch (signupError) {
-      throw new Error('Something went wrong. Please try again later');
+    } catch (errorData: any) {
+      setServerError(errorData?.message);
     }
   };
 
-  return <AuthForm onSubmit={onSubmit} />;
+  return <AuthForm onSubmit={onSubmit} error={serverError} />;
```

Comment:
i don't like this approach, about onSubmit it looks good but with error, looks complicated, I think we need to try simplify this case

Reply:
I don't have a good solution to this problem, perhaps only a special hook can be made.